protobuf/grpc_service.proto

```diff
@@ -462,6 +462,18 @@ message InferParameter
     //@@       A string parameter value.
     //@@
     string string_param = 3;
+
+    //@@    .. cpp:var:: double double_param
+    //@@
+    //@@       A double parameter value.
+    //@@
+    double double_param = 4;
```

Comment:
I think it might be better to add this parameter when `double` is added to the kserve protocol. Looks like currently it is not being used and we don't have any testing for it so I think it would be better to remove it for now.

Reply:
I was thinking the same, but wasn't sure if it was explicitly agreed to be added from kserve meeting or not.

Reply:
it is agreed to be added - I'm fine to remove it until agreed upon - only thing is we will keep the uint64 at # 5 to avoid conflict. Overall preference?

Reply:
> only thing is we will keep the uint64 at # 5 to avoid conflict.
conflict with who? Are double/uint64 already defined as 4/5 in kserve?

I don't see it in kserve/open-inference-protocol repos. Can you share link to conflict?

Reply:
https://github.com/Xilinx/inference-server/blob/663098bc7320d345fc3676aa8b7d361814352c37/src/amdinfer/core/inference.proto#L17

I plan to make a consolidated change to KServe after our changes are merged